### PR TITLE
Updated validation code for ASB

### PR DESCRIPTION
**Pull request by kollil**
Updated ASB procedures validation

### CODE REVIEW

**Review by Ohsudev — APPROVED** [review]

File: onprc_ehr/resources/scripts/onprc_ehr/onprc_triggers.js [hunk 1]
```diff
@@ -238,81 +238,90 @@ exports.init = function(EHR){
...
+        // “WARN: If entering Remarks, you must select “Other…” in ASB Special Instructions Field” and grey out the “Request” button until the above action is done.
+
+        // 2. If a user selects any procedure, then chooses "None" within "ASB Special Instructions" and continues to add a remark in the remarks section,
+        // add the flag for the “None” option: "If choosing "None", you must leave "Remarks" field blank"
+
+        //New code added to make this validation work only for the ASB form, Kollil on 10/05/21
+        helper.decodeExtraContextProperty('ASBRequestForm', false);
+        var inASBRequestForm = helper.getProperty('ASBRequestForm');
+
+        console.log(" inASBRequestForm: " + inASBRequestForm);
```
Comment (labkey-jeckels) [inline onprc_ehr/resources/scripts/onprc_ehr/onprc_triggers.js]: I recommend deleting this logging code. It's useful for troubleshooting on dev machines but will clutter the log file on production.

File: onprc_ehr/resources/scripts/onprc_ehr/onprc_triggers.js [hunk 2]
```diff
@@ -238,81 +238,90 @@ exports.init = function(EHR){
...
+        if (inASBRequestForm == null) {
+            console.log("In");
```
Comment (labkey-jeckels) [inline onprc_ehr/resources/scripts/onprc_ehr/onprc_triggers.js]: I recommend deleting this logging code. It's useful for troubleshooting on dev machines but will clutter the log file on production.

**Review by labkey-jeckels — APPROVED** [review]
Suggested logging cleanup but otherwise looks good.

File: onprc_ehr/resources/scripts/onprc_ehr/onprc_triggers.js (hunk 1 above)
Comment (kollil) [reply onprc_ehr/resources/scripts/onprc_ehr/onprc_triggers.js]: Thanks Josh, I removed the log statements.

**Pull request merged**